src/Centreon/Application/ApiPlatform.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * Copyright 2005 - 2020 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+declare(strict_types=1);
+
+namespace Centreon\Application;
+
+/**
+ * Used to get API parameters
+ */
+class ApiPlatform
+{
+    /**
+     * @var float
+     */
+    private $version;
+
+    /**
+     * Get the API version
+     *
+     * @return float
+     */
+    public function getVersion(): float
```

Comment:
this couldn't be nullable ?

Reply:
I don't think so. As the APi version is mandatory and if it's missing a 403 Forbidden is returned